app.json

```diff
@@ -49,7 +49,7 @@
       "required": true
     },
     "HOSTNAME": {
-      "description": "Hostname of your web application.",
+      "description": "Hostname of your web application."
```

Comment:
This was a bug in the app.json file that needed to be fixed for proper deployment of the app to Heroku